Modules/core/BRPeer.c

```diff
@@ -58,8 +58,8 @@
 #define MAX_MSG_LENGTH     0x02000000
 #define MAX_GETDATA_HASHES 50000
 #define ENABLED_SERVICES   0ULL  // we don't provide full blocks to remote nodes
-#define PROTOCOL_VERSION   70025
-#define MIN_PROTO_VERSION  70017 // peers earlier than this protocol version not supported (need v0.9 txFee relay rules)
+#define PROTOCOL_VERSION   70027
+#define MIN_PROTO_VERSION  70026 // peers earlier than this protocol version not supported (need v0.9 txFee relay rules)
```

Comment:
Verified this is the version in the current release: https://github.com/RavenProject/Ravencoin/blob/release_3.3.1/src/version.h#L13